playbooks/roles/load-balancer-v2/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+
+# TODO: add the rest
+# TODO: add HAProxy installation
+# TODO: add installation of script that wraps consul-template calls
+
+- name: Copy over consul-template template files
+  copy:
+    src: "{{ item }}"
+    dest: "{{ consul_template_tmpl_dir }}/{{ item }}"
+  with_items:
+    - backend.map.ctmpl
+    - certs.pem.ctmpl
+    - haproxy.cfg.ctmpl
+
+- name: Copy over consul-template configuration files
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ consul_template_config_dir }}/{{ item }}"
+  with_items:
+    - backend.map.hcl
+    - certs.pem.hcl
+    - haproxy.cfg.hcl
```

Comment:
I didn't see these files on haproxy-c-1.net.opencraft.hosting -- I guess this is because you didn't run this role on that host yet, but I am noting it just in case.

Reply:
Yep, didn't run this, as it's not complete - just used the tag.